charts/atlas-cluster/values.yaml

```diff
@@ -60,7 +60,7 @@ clusters:
 users:
   - username: admin-user
     databaseName: admin
-    password: "%SomeLong%password$foradmin"
+    password: 
```

Comment:
Maybe just remove this attribute from the file, makes it harder for users to mistakenly commit it or share